.github/workflows/call-spell-check.yml

```diff
@@ -0,0 +1,15 @@
+# Runs devtools::spell_check().
+name: call-spell-check
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
```

Comment:
I think will lead to two workflows on the same code so I recommend removing lines 8--10.